src/main/scala/nl/knaw/dans/lib/logging/servlet/masked/Masker.scala

```diff
@@ -39,11 +39,18 @@ object Masker {
    * Services without public access might not need to mask.
    */
   def formatRemoteAddress(remoteAddress: String): String = {
-    // TODO https://docs.oracle.com/javase/9/docs/api/java/net/Inet6Address.html
-    // see also:
-    //  * https://stackoverflow.com/questions/3118829/why-request-getremoteaddr-returns-ipv4-or-ipv6-depending-on-context-post-quer
-    //  * https://stackoverflow.com/questions/7589526/what-is-the-ip-format-returned-by-servletrequest-getremoteaddr
-    remoteAddress.replaceAll("([0-9]+[.]){3}", "**.**.**.")
+    remoteAddress match {
```

Comment:
can you identify for each of the cases (except for the first one, of course), what kind of remote addresses you're expecting. A comment like `// IPv4`, `// IPv6`, etc. would do great here!